backend/config/locales/zu.yml

```diff
@@ -192,6 +198,12 @@ zu:
     rejected:
       subject: Your account was rejected!
       content_html: We would like to inform you that your account at HeCo platform was rejected. One of our platform administrators should reach you shortly, but feel free to contact us at %{contact_url}.
+  activerecord:
```

Comment:
Hmm, should we keep consistency with the existing forms and use `simple_forms` namespace?

Reply:
Actually, this will deal with error messages as well, right? And simple_forms will not. Maybe then we should move all simple_forms to attributes, even if those labels are long and not named as attributes, but not in this PR. I would have to check what will happen with existing translations when we change key names but not value, maybe Transifex is intelligent about that.

Reply:
Oki douky, I will move new text under `simple_form` prefix for now and we can refactor it later within separate PR ;)

Reply:
I find the possible aftermath terrifying :) I don't have high hopes for anything working automatically, but I'm not sure. Perhaps rather than renaming we could copy the strings under the new namespace, that will at least make it easier to copy existing translations rather than losing them and starting from scratch?

Reply:
I think I did similar things very long time ago, and it worked, but who knows what have changed. We already know that Transifex CLI is not very friendly.